ynput/OpenPype
# Max: Bug fix the resolution not being shown correctly in review burnin

## Changelog Description
The resolution is not being shown correctly in review burnin
## Additional info
n/a

## Testing notes:
1. Launch Max via launcher
2. Go to Scripting -> MaxScript Listener
3.Type `viewport.setLayout #layout_2v`
4. Create Review
5. Publish
6. Make layout similar to the screenshot 
![image](https://github.com/ynput/OpenPype/assets/64118225/5db6c9af-3c61-4931-8acf-56c6b5cb8eb5)

7. Publish with the existing review instance
8. Compare Two burnin of which is published respectively after 3 and 6, they should be at the same resolution without any stretch.
